understanding/understanding-template.html

```diff
@@ -37,7 +37,7 @@ <h2>Resources</h2>
 		<section id="techniques">
 			<h2>Techniques</h2>
 			<div class="instructions">
-				<p>This section references techniques that can be used to meet the Guideline or Success Criterion. There are sub-sections for sufficient techniques, advisory techniques, and failures. </p>
+				<p>This section references techniques that can be used to meet the guideline or success criterion. There are sub-sections for sufficient techniques, advisory techniques, and failures. </p>
```

Comment:
As per my prior comment, I'm inclined to leave all the G's capitalized in Guideline, when used in this sense. Don't feel super strong about it though.

Reply:
OK by me